src/popup/components/MultisigVaultCreateReview.vue

```diff
@@ -146,12 +148,30 @@ export default defineComponent({
     const creatorAccount = computed(
       () => accounts.value.find(({ address }) => address === creatorAddress.value),
     );
+    const fee = computed(() => multisigAccountCreationFee.value);
+    const callData = computed(() => multisigAccountCreationEncodedCallData.value);
 
     watch(creatorAddress, async (val, oldVal) => {
       if (val !== oldVal) {
         creatorAccountFetched.value = undefined;
         const sdk = await getSdk();
         creatorAccountFetched.value = await sdk.api.getAccountByPubkey(val) as IAccountFetched;
+        const { idx: activeAccountIdx } = account.value;
+        root.$store.commit(
+          'accounts/setActiveIdx',
+          accounts.value.find(({ address }) => address === val)?.idx,
```

Comment:
Shouldn't we break this function execution in case there is no matching account found?

Reply:
It is guranteed to be found because we are watching here for account selection and  that list is from `acccounts` state itself.